Libraries/LibWeb/HTML/HTMLTitleElement.cpp

```diff
@@ -50,4 +58,35 @@ void HTMLTitleElement::set_text(String const& value)
     string_replace_all(value);
 }
 
+void HTMLTitleElement::consider_propagate_title_update()
+{
+    if (text().is_empty()) {
+        m_throttle_update_timer->restart();
```

Comment:
A title update is now delayed indefinitely if you keep setting it to an empty string quickly enough... instead of `restart()`, just call `start()` here.